server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -798,6 +800,18 @@ public int getNodesCountInGivenLevel(String prefixPath, int level) throws Metada
           }
         }
       }
+
+      // if ordered by heat, we sort all the timeseries by the descending order of the last insert timestamp
+      if (plan.isOrderByHeat()) {
+        allMatchedNodes = allMatchedNodes.stream().sorted(
+            Comparator.comparingLong(MTree::getLastTimeStamp).reversed()
+                .thenComparing(MNode::getFullPath)).collect(toList());
```

Comment:
do we need to order by path?

Reply:
@qiaojialin Hi, in my opinion, order by path is better than disorder relatively.